### DISCUSSION TITLE
Files viewed can show incorrect number

### DISCUSSION THREAD

**Discussion by mnizamud** [community/community#23747] [New to GitHub]
<p>When going through PR’s, I’ve notice that if I’ve asked to remove a change from a file, and after the next commit the file no longer has a change, the “Files Viewed” doesn’t account for the file that no longer exists in the PR.  So I get something that looks like the attached image below.<br>
It would be really nice if this were to account for the files which no longer have a change and were removed from the PR.</p>
<p><img src="https://user-images.githubusercontent.com/57022111/181079218-22822b8c-5b50-4554-ac52-8a7bf47bb7b8.png" alt="Screen Shot 2020-06-16 at 09.35.59" data-base62-sha1="MNIDNHst3v1uR6RtwewWErRMc2" width="354" height="104"></p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>This is a known issue, and the engineering team is working on it! Thanks for taking the time to pass on your feedback!</p>

**Comment by pecasdev** [comment]
Just found this issue now and wanted to reply that the problem is still present
![image](https://github.com/community/community/assets/33271004/9bda5218-d06b-4379-9258-7080284fa71d)
